packages/kit/src/core/build/index.js

```diff
@@ -77,6 +78,25 @@ export async function build(config, { cwd = process.cwd(), runtime = '@sveltejs/
 	};
 }
 
+/**
+ * @param {string?} scope used to prefix the error message
+ * @param {string[]} conflicts array of conflicts in dotted notation
+ */
+function print_vite_config_conflicts(scope, conflicts) {
```

Comment:
do we want this to happen only on `build` or should it happen for `dev` as well?

Reply:
Oh yeah, I was focused on the rollup option, which aren't applicable to dev. I updated to also do dev.